### PR TITLE
Updates for about page

### PR DESCRIPTION
**Pull request by sowasred2012**
## Done

Copy update on `/about`

## QA

- Go to `/about` or https://juju-is-399.demos.haus/about
- [Copy doc](https://docs.google.com/document/d/186zQ7bYE-jXqNNTyKEvSzjq-yP-MiMW2zHKbgyrBrm4/edit) 

## Issue / Card

Fixes [#5270](https://github.com/canonical-web-and-design/web-squad/issues/5270)

### CODE REVIEW

**Comment by spencerbygraves** [comment]
Thanks @sowasred2012, I think the 3 card titles would be better as h4. Thanks.

File: templates/about/_practices_tabs.html [hunk 1]
```diff
@@ -74,7 +74,7 @@ <h3 class="u-hide--large">Excellent communities build excellent user experience<
...
       
-      <p>What would this allow you to do? Could you focus on the logic - on the business decisions - that give meaning to those operations? Together, we can combine the knowledge of the open source community into shared, open source applications, which are coded to react to different scenarios, situations, and operations as you’d like them to.</p>
+      <p>What would this allow you to do? Could you focus on the logic - on the business decisions - that give meaning to those operations? Together, we can combine the knowledge of an open source expert community into shared, open source applications, which are coded to react to different scenarios, situations, and operations as you’d like them to.</p>
```
Comment (mtruj013) [inline templates/about/_practices_tabs.html:77]:
```suggestion
      <p>What would this allow you to do? Could you focus on the logic &ndash; on the business decisions &ndash; that give meaning to those operations? Together, we can combine the knowledge of an open source expert community into shared, open source applications, which are coded to react to different scenarios, situations, and operations as you’d like them to.</p>
```

**Comment by tmihoc** [comment]
In the Technology table, if possible, please try to align the paragraphs vertically as in the copy doc. Please also bold the names of the components, as in the copy doc.

**Comment by anthonydillon** [comment]
> Just noticed that in the Mission section the text font size varies---large in the first paragraph and smaller in the other two paragraphs. Please make it all the same, as in the copy doc.

Working on this now, no problem.

**Comment by anthonydillon** [comment]
> Also, the intro text right under About Juju has short line width. Please make it full line width, as in the copy doc.

This max with is hard set within the typographic system of Vanilla. I can override it but it would make the text less legible.

**Comment by anthonydillon** [comment]
> Just noticed that in the Mission section the text font size varies---large in the first paragraph and smaller in the other two paragraphs. Please make it all the same, as in the copy doc.

This is done as part of this PR: https://github.com/canonical-web-and-design/juju.is/pull/401

**Pull request merged**